app/assets/javascripts/editor/editor.js.erb

```diff
@@ -1043,6 +1058,15 @@ var CodeOceanEditor = {
           delta_data.text === last_delta.text;
     },
 
+    showUserIsOnline: function (status_and_user) {
+        if (status_and_user.status === 'connected') {
+            $('#pg_session').text(status_and_user.current_user_name + I18n.t('exercises.editor.is_online'));
```

Comment:
What if... we change the translation, so that it already allows passing a user name?

```suggestion
            $('#pg_session').text(I18n.t('exercises.editor.is_online', {name: status_and_user.current_user_name}));
```

This, of course, also requires a corresponding change in the en.yml / de.yml.